go.mod

```diff
@@ -8,12 +8,10 @@ require (
 	github.com/devfile/library/v2 v2.2.2-0.20240108233338-2a1d045e1e65
 	github.com/go-logr/logr v1.4.1
 	github.com/gofri/go-github-ratelimit v1.0.3-0.20230428184158-a500e14de53f
-	github.com/golang/mock v1.6.0
```

Comment:
it seems like the module `github.com/redhat-developer/gitops-generator` is still in go.mod

Reply:
We still use it for some unrelated error sanitation. It was easier to leave it, than to refactor it out.